DESCRIPTION

```diff
@@ -36,5 +37,7 @@ Suggests:
     rmarkdown,
     sf,
     tibble
+remotes:
+    r-lib/tidyselect
 VignetteBuilder: knitr
```

Comment:
and I think we can remove this now that 1.0.0 is on CRAN

Reply:
done